lib/types/luks.nix

```diff
@@ -114,19 +114,23 @@ in
       default = ''
         if ! blkid "${config.device}" >/dev/null || ! (blkid "${config.device}" -o export | grep -q '^TYPE='); then
           ${lib.optionalString config.askPassword ''
-            set +x
             askPassword() {
-              echo "Enter password for ${config.device}: "
-              IFS= read -r -s password
-              echo "Enter password for ${config.device} again to be safe: "
-              IFS= read -r -s password_check
-              export password
-              [ "$password" = "$password_check" ]
+              if [ -z ''${IN_DISKO_TEST+x} ]; then
```

Comment:
Why is this needed? To an interactive password code path as opposed to just key files?

I think it can be a bit smelly to let code check whether it's in a test here 😅

Reply:
Can we not just override passwordFile in disko?

Reply:
no, that's not possible since disko doesn't merge the devices correctly

Reply:
Can we set a "default" with higher than normal priority through "options" instead?

Reply:
no, since we can't modify the the options after we evaluate the vm attribute, because the config is already evaluated